scenarios/json.benchmarks.yml

```diff
@@ -79,3 +92,16 @@ scenarios:
       variables:
         path: /json
         presetHeaders: json
+
+  json_400k:
+    application:
+      job: jsonbenchmarks
+      variables:
+        scenario: json
+      environmentVariables:
+        JSONSIZE: 400000
```

Comment:
FYI could be a command line argument plus variable and have a default value. Something like "connections" in client scenarios.